x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/types.ts

```diff
@@ -41,29 +45,41 @@ export interface AddRulesProps {
   signal: AbortSignal;
 }
 
+const MetaRule = t.type({
+  from: t.string,
+});
+
 export const RuleSchema = t.intersection([
   t.type({
+    created_at: t.string,
```

Comment:
No need to fix here/now as we're moving to API Import/Exporting, but wanted to comment that importing is now not functional as a result of these type changes.

<img src="https://user-images.githubusercontent.com/2946766/71211948-7d77bc00-226d-11ea-97f0-2636d6c425b9.png" width=300>